test/test_builders.py

```diff
@@ -52,6 +62,12 @@ def test_entity_to_vitessce_conf(entity_path, mocker):
     # Need to mock zarr.open to yield the correct error with an empty store
     mocker.patch('zarr.open', return_value=zarr.open())
 
+    possible_gene = entity_path.name.split('-')[-2]
+    gene = (
+        possible_gene.split('=')[1]
+        if possible_gene.startswith('gene=')
+        else None)
+
```

Comment:
It is hacky... thought this would be more readable than a regex... but until we have multiple parameters to pass in, I didn't want to over-engineer it... If we have different values coming in, can revisit.